src/voting-metadata-api/src/models/errors/httpResponseException.ts

```diff
@@ -21,3 +21,9 @@ export class ForbiddenException extends HTTPResponseException {
     super(403, message)
   }
 }
+
+export class ServiceException extends HTTPResponseException {
+  constructor(message: string) {
+    super(500, message)
```

Comment:
Would this leak the message to the client? If so, is that a security risk in case the error has internal info?

Reply:
It didn't look like `tsoa` emitted the message to the client but it's a good idea to remove just in case